sdk/eventhub/event-hubs/src/linkEntity.ts

```diff
@@ -237,15 +238,15 @@ export class LinkEntity {
         // Closing the link and its underlying session if the link is open. This should also
         // remove them from the internal map.
         await link.close();
-        log.link(
+        log.info(
           "[%s] %s '%s' with address '%s' closed.",
           this._context.connectionId,
           this._type,
           this.name,
           this.address
         );
       } catch (err) {
-        log.error(
+        log.verbose(
```

Comment:
Looks like we don't throw an error that can occur when closing link, but we do have a catch and re-throw happening in `EventHubClient.close()` ...